webapp/locales/de.json

```diff
@@ -235,7 +236,9 @@
       "postCount": "Beiträge"
     },
     "tags": {
-      "name": "Schlagworte",
+      "name": "Hashtags",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-12T15:02:10Z" title="Monday, August 12th 2019, 5:02:10 pm +02:00">Aug 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -235,7 +236,9 @@
       "postCount": "Beiträge"
     },
     "tags": {
-      "name": "Schlagworte",
+      "name": "Hashtags",
```


I prefer "Schlagworte" or "Schlagwörter".

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-12T18:00:30Z" title="Monday, August 12th 2019, 8:00:30 pm +02:00">Aug 12, 2019</time>_

---


But hashtags is used all over the place in sozial networks @roschaefer !?
Even on german ones …